exporter/importables.go

```diff
@@ -2884,10 +2916,7 @@ var resourcesMap map[string]importable = map[string]importable{
 			return nil
 		},
 		Import: func(ic *importContext, r *resource) error {
-			ic.Emit(&resource{
-				Resource: "databricks_grants",
-				ID:       "metastore/" + r.ID,
-			})
+			ic.emitUCGrantsWithOwner("metastore/"+r.ID, r)
```

Comment:
I would export the metastore without the owner, otherwise you cannot create grants on objects you do not own

Reply:
Really, on the workspace level we won't emit metastores at all (the next PR is coming for it).